### PR TITLE
Mobile: update embed message and embed component mobile wrap components

### PR DESCRIPTION
**Pull request by hoangtranlehuy**
Mobile: update embed message and embed component mobile wrap components
https://github.com/nccasia/mezon/issues/4979

### CODE REVIEW

**Comment by hoangtranlehuy** [comment]
Screenshot 2024-11-21 161702

**Comment by hoangtranlehuy** [comment]
image

File: apps/mobile/src/app/screens/home/homedrawer/components/EmbedMessage/EmbedTitle/index.tsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+	url: string;
+}
+
+export const EmbedTitle = memo(({ title, url }: EmbedDescriptionProps) => {
+	const { themeValue } = useTheme();
+	const styles = style(themeValue);
+
+	const handleOpenUrl = async () => {
+		const store = await getStoreAsync();
+		store.dispatch(appActions.setLoadingMainMobile(true));
```
Comment (phuocnguyenncc) [inline apps/mobile/src/app/screens/home/homedrawer/components/EmbedMessage/EmbedTitle/index.tsx]: remove loading

File: apps/mobile/src/app/screens/home/homedrawer/components/EmbedComponents/EmbedComponentItem/index.tsx [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+			case EButtonMessageStyle.SECONDARY:
+				return themeValue.bgViolet;
+			case EButtonMessageStyle.SUCCESS:
+				return themeValue.bgViolet;
+			case EButtonMessageStyle.DANGER:
+				return themeValue.bgViolet;
+			case EButtonMessageStyle.LINK:
+				return themeValue.bgViolet;
+			default:
+				return themeValue.bgViolet;
```
Comment (phuocnguyenncc) [inline apps/mobile/src/app/screens/home/homedrawer/components/EmbedComponents/EmbedComponentItem/index.tsx]: check switch color

File: apps/mobile/src/app/screens/home/homedrawer/MessageItem.tsx [hunk 3]
```diff
@@ -340,7 +341,18 @@ const MessageItem = React.memo(
 										onLongPress={handleLongPressMessage}
 									/>
 								)}
-								{!!message?.content?.embed && <EmbedMessage {...message.content.embed} />}
+								{!!message?.content?.embed?.length &&
+									message?.content?.embed?.map((embed, index) => <EmbedMessage {...embed} key={index} />)}
```
Comment (phuocnguyenncc) [inline apps/mobile/src/app/screens/home/homedrawer/MessageItem.tsx]: rename key

File: apps/mobile/src/app/screens/home/homedrawer/MessageItem.tsx [hunk 4]
```diff
@@ -340,7 +341,18 @@ const MessageItem = React.memo(
...
+								{!!message?.content?.components?.length &&
+									message?.content.components?.map((component) => (
+										<EmbedComponentsPanel
+											key={message?.id}
```
Comment (phuocnguyenncc) [inline apps/mobile/src/app/screens/home/homedrawer/MessageItem.tsx]: same

File: apps/mobile/src/app/screens/home/homedrawer/MessageItem.tsx (hunk 3 above)
Comment (phuocnguyenncc) [reply apps/mobile/src/app/screens/home/homedrawer/MessageItem.tsx]: message_embe_${mes}_${index}

File: apps/mobile/src/app/screens/home/homedrawer/components/EmbedMessage/EmbedTitle/index.tsx (hunk 1 above)
Comment (phuocnguyenncc) [reply apps/mobile/src/app/screens/home/homedrawer/components/EmbedMessage/EmbedTitle/index.tsx]: add try catch

**Review by phuocnguyenncc — APPROVED** [review]

**Pull request merged**